### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by khoslaventures**
- Add genesis running guide
- Update some broken commands and details on the original gentx guide

### CODE REVIEW

File: olympus_mons/run.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+## GenTx submissions are done and closed!
+
+To see if you were included in the final genesis, you can search `genesis.json` or the `final-gentxs` folder for your `moniker`, `delegation_address` or other identifiers unique to you.
+
+If you were included, congrats! there were a lot of submissions to process and we're glad to have you.
+
+If you weren't, thank you for taking the time to submit - we suggest looking at the [Mars Meteor Missions](https://evmos.blog/evmos-incentivized-testnet-event-the-mars-meteor-missions-bbbb7ffa1b7c) for participating in the Olympus Mons incentivzed testnet as the rewards can end up much higher for the other missions!
+
+We encourage you to find ways to jail validators if you're really interested in becoming part of the 300. Once the network launches.
```
Comment (khoslaventures) [inline olympus_mons/run.md:11]: @fedekunze @jolube thoughts?

**Review by jolube — APPROVED** [review]

File: olympus_mons/run.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
+# Running as a genesis validator
+
+## GenTx submissions are done and closed!
+
+To see if you were included in the final genesis, you can search `genesis.json` or the `final-gentxs` folder for your `moniker`, `delegation_address` or other identifiers unique to you.
+
+If you were included, congrats! there were a lot of submissions to process and we're glad to have you.
+
+If you weren't, thank you for taking the time to submit - we suggest looking at the [Mars Meteor Missions](https://evmos.blog/evmos-incentivized-testnet-event-the-mars-meteor-missions-bbbb7ffa1b7c) for participating in the Olympus Mons incentivzed testnet as the rewards can end up much higher for the other missions!
```
Comment (fedekunze) [inline olympus_mons/run.md:9]:
```suggestion
If you weren't, thank you for taking the time to submit - we suggest looking at the [Mars Meteor Missions](https://evmos.blog/evmos-incentivized-testnet-event-the-mars-meteor-missions-bbbb7ffa1b7c) for participating in the Olympus Mons incentivized testnet as the rewards can end up much higher for the other missions!
```

File: olympus_mons/run.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+We encourage you to find ways to jail validators if you're really interested in becoming part of the 300 once the network launches, subject to our [Code of Conduct](https://www.notion.so/tharsis/Code-of-Conduct-for-Evmos-Incentivized-Testnet-802fc5298ef647ca954c5dc0d44d39c1).
```
Comment (fedekunze) [inline olympus_mons/run.md:11]:
```suggestion
We encourage you to find ways to jail validators if you're really interested in becoming part of the active validator set once the network launches, subject to our [Code of Conduct](https://www.notion.so/tharsis/Code-of-Conduct-for-Evmos-Incentivized-Testnet-802fc5298ef647ca954c5dc0d44d39c1).
```

File: olympus_mons/run.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+## For selected validators
+**The Chain Genesis Time is 19:00 UTC on Nov 25, 2021.**
```
Comment (fedekunze) [inline olympus_mons/run.md:13]:
```suggestion

**The Chain Genesis Time is 19:00 UTC on Nov 25, 2021.**
```

File: olympus_mons/run.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+Please have your validator up and ready by this time, and be available for further instructions if necessary
+at that time.
+
+The primary point of communication for the genesis process will be the #validators
+channel on the [Evmos Discord](https://discord.gg/PJeeUYRHuY). It is absolutely critical that you
```
Comment (fedekunze) [inline olympus_mons/run.md:19]: Does this link expire?

**Review by fedekunze — APPROVED** [review]
I think the installation and cosmovisor instructions should link to our docs instead tho

**Pull request merged**